package.json

```diff
@@ -33,6 +33,8 @@
     "@types/node": "^20.10.3",
     "@vitest/coverage-v8": "^1.0.3",
     "changelogen": "^0.5.5",
+    "detect-indent": "^7.0.1",
+    "detect-newline": "^4.0.1",
```

Comment:
Would be nice to avoid exta deps and use simple utility to do the same (also these are dev dependencies now)

Reply:
Good call, I made a detectNewline util and removed the dependency. And also moved the detect-indent package from dev dependencies to dependencies.

Reply:
Hi, @pi0. Would be great to get this merged because every time we run changelogen release we have to use the `--no-commit` flag and then run `lint --fix` because the package.json fails our lint check. Lmk if theres anything else I can do.